README.md

```diff
@@ -26,13 +25,26 @@ No providers.
 | Name | Version |
 |------|---------|
 | <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | ~> 1.0 |
+| <a name="requirement_github"></a> [github](#requirement\_github) | >= 4.19.1 |
 
 <!-- TFDOCS_REQUIREMENTS_END -->
 
 <!-- TFDOCS_INPUTS_START -->
 ## Required Inputs
 
-No required inputs.
+The following input variables are required:
+
+### <a name="input_token"></a> [token](#input\_token)
```

Comment:
I don't like the seemingly standard "input_" prefix for every variable.
It's probably coming from the docs generator but I don't subscribe to this standard.
Taking this one notch further, we can also rename the variable to actually have its name a bit more suggestive.

Reply:
@danvaida this is just an reference which terraform-docs generate. check variable definition - it is `token`

Reply:
I know. I just like to have my documentation matching the name of the vars and outputs one-to-one, w/o any other decorations.
But that's just me...